test/functional/test_framework/test_framework.py

```diff
@@ -1122,8 +1157,17 @@ def activate_dip0024(self, expected_activation_height=None):
     def activate_v19(self, expected_activation_height=None):
         self.activate_by_name('v19', expected_activation_height)
 
-    def activate_v20(self, expected_activation_height=None):
-        self.activate_by_name('v20', expected_activation_height)
+    def activate_v20(self, expected_activation_height=None, test_locked_in_phase=False):
+        if test_locked_in_phase:
+            # Expected locked_in phase starts at 1440 - 480 (window size in regtest) + 1
+            projected_activation_height = self.advance_to_locked_in_softfork('v20', expected_locked_in_height=961)
+            self.activate_by_name('v20', expected_activation_height)
+            softfork_info = get_bip9_details(self.nodes[0], 'v20')
+            assert softfork_info['status'] == 'active'
```

Comment:
```suggestion
            assert_equal(softfork_info['status'], 'active')
```

assert_equal will log to screen values of 2 arguments, the output will be something like:
```
assert_equal failed at line X because 'defined' is not equal to 'active'.
```